WordPress/src/main/res/values/styles.xml

```diff
@@ -1585,27 +1589,19 @@
 
     <style name="ThemeActionSheetTitle">
         <item name="android:fontFamily">serif</item>
-        <item name="android:textColor">@color/title_color</item>
+        <item name="android:textColor">?attr/colorOnSurface</item>
         <item name="android:textSize">@dimen/theme_bottom_sheet_title_text_size</item>
         <item name="android:textStyle">bold</item>
     </style>
 
     <style name="ThemeActionSheetOptionText">
-        <item name="android:textColor">@color/title_color</item>
+        <item name="android:textColor">?attr/colorOnSurface</item>
         <item name="android:textSize">@dimen/theme_bottom_sheet_text_size</item>
     </style>
 
-    <style name="ThemeBottomSheetButtonFilled" parent="Widget.MaterialComponents.Button">
+    <style name="Widget.ModalLayoutPicker.Button.Primary.accent" parent="Widget.MaterialComponents.Button.UnelevatedButton">
+        <item name="android:textAppearance">?attr/textAppearanceSubtitle2</item>
+        <item name="android:textColor">?attr/colorOnSecondary</item>
         <item name="android:backgroundTint">@color/accent</item>
-        <item name="android:textAllCaps">false</item>
-        <item name="android:textColor">@color/white</item>
-        <item name="cornerRadius">@dimen/theme_bottom_sheet_button_corner_radius</item>
-    </style>
-
-    <style name="ThemeBottomSheetButtonOutlined" parent="Widget.MaterialComponents.Button.OutlinedButton">
```

Comment:
For the same reason as [above](https://github.com/wordpress-mobile/WordPress-Android/pull/15029/commits/f07ac7c17c8736c672810d1658db2e55e7214bc9#r673927549) I would keep the `ThemeBottomSheetButtonOutlined` style (to use it in the button) and set as parent the `Widget.ModalLayoutPicker.Button.Secondary` if didn't want to set the same attributes again.